docs/datastore.md

```diff
@@ -0,0 +1,94 @@
+
+- Users
+- EmploymentContracts
+- Reminders
+
+### Users
+
+The composition of a Users document is:
+
+| Field | Datatype | Description |
+| :------------- |:-------------:| -----:|
+| _id     | String | The unique identifier of a user. We use Expo.Constants.deviceId for this |
+
+### EmploymentContracts
+
+The composition of a EmploymentContracts document is:
+
+| Field | Datatype | Description |
+| :------------- |:-------------:| -----:|
+| _id     | String | The unique identifier of an employment contract. |
+| user_id     | String | The unique identifier of user who owns the employment contract. |
+| name    | String | The title of the employment contract |
+| startDate_exact     | Date | If an exact start date is known this is the suiting field  |
+| startDate_string    | String | If no exact start date is known this is the suiting field  |
+| end_exact     | Date | If an exact end date is known this is the suiting field  |
+| endDate_string    | String | If no exact end date is known this is the suiting field  |
+
+### Reminders
+
+The compostion of a Reminders document is:
+
+| Field | Datatype | Description |
+| :------------- |:-------------:| -----:|
+| _id     | String | The unique identifier of a reminder. |
+| user_id     | String | The unique identifier of user who owns the reminder. |
+| description    | String | A description for the reminder  |
+| date    | Date | The date the user should be reminded  |
+
+
+## How to access the datastore through code
+
+In contrast to the previous interation of MongoDB in our project, the database can no longer be accessed directly through Rest. The only way to access the database now, is through some services in the backend. They all are part of `backend/src/database`:
+
+| Document-Type | Path to the Service | 
+| :------------- |-------------:| 
+| Users    | `backend/src/database/user/user.service.ts` | 
+| EmploymentContracts   | `backend/src/database/employmentContract/employmentContract.service.ts` | 
+| Reminders  | `backend/src/database/reminder/reminder.service.ts` | 
+
+To access the datastore import the suiting service and create an instance of it. After that you can create new database entries, modify existing ones and delete entries. For a full list of functions and their meaning take a look at the corresponding JsDoc. Let's include a short example of creating a new reminder, reading all reminders for a specific user at a specific date and delete a reminder:
+
+```javascript
+// Import the suiting service
+import { ReminderService } from 'backend/src/database/reminder/reminder.service.ts';
+
+class ReminderDemo {
+    private reminderService: ReminderService;
+
+    constructor(reminderService: ReminderService) {
+        this.reminderService = reminderService;
+    }
+
+    public demo() { 
+      // Create a new reminder
+      let id = await reminderSer.create("John Doe", "This is a description", new Date(1995,6,23));
```

Comment:
Hier ist noch ein winziger Fehler in der Doku. Es müsste this.reminderService sein. (Weiter unten ebenfalls)